### PR TITLE
Did basic toggle color theme for the app.

### LINKED ISSUE #1: Tic-tac-toe Dark mode implementation
Some users have pointed out that having a dark mode implemented in my tic tac toe would be very beneficial for them.
- [ ] User should be able to switch from light to dark mode
- [ ] Dark mode should have a lower overall brightness and enough contrast, that way users can use it in low ambient light conditions

### PR DESCRIPTION
**Pull request by jzhupan**
Did basic toggle color theme with a button, using useState and useEffect.

### CODE REVIEW

File: src/public/index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
```
Comment (jcari-dev) [inline src/public/index.html]: Meta tags are not self closing

File: src/public/index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (jcari-dev) [inline src/public/index.html]: meta tags are not self closing for more info see https://www.w3schools.com/tags/tag_meta.asp

File: src/styles.css [hunk 3]
```diff
@@ -73,7 +73,7 @@ body {
 
 .board-row:after {
   clear: both;
-  content: '';
+  content: "";
```
Comment (jcari-dev) [inline src/styles.css]: Keep the single quotes. For more info check the Google CSS Styling Guide: https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks

**Review by jcari-dev — CHANGES_REQUESTED** [review]
Please check the comments, if possible read/ check the HTML/CSS Google style guide.

File: src/styles.css (hunk 3 above)
Comment (jzhupan) [reply src/styles.css]: I see, thank you so much for the resources!

File: src/public/index.html (hunk 1 above)
Comment (jzhupan) [reply src/public/index.html]: Changes made.

**Review by jcari-dev — APPROVED** [review]
LGTM!

**Pull request merged**